packages/playwright-core/src/server/registry/browserFetcher.ts

```diff
@@ -90,6 +95,20 @@ function downloadBrowserWithProgressBarOutOfProcess(title: string, browserDirect
   cp.on('error', error => {
     promise.resolve({ error });
   });
+
+  debugLogger.log('install', `running download:`);
+  debugLogger.log('install', `-- from url: ${url}`);
+  debugLogger.log('install', `-- to location: ${zipPath}`);
+  const downloadParams: DownloadParams = {
+    title,
+    browserDirectory,
+    url,
+    zipPath,
+    executablePath,
```

Comment:
IIRC, passing `undefined` through IPC turns it into `null`? Let's make sure nothing breaks in this case.

Reply:
I would think this happens only to the top-level value.